src/datepicker/datepicker.js

```diff
@@ -138,58 +139,72 @@ export default class Datepicker extends React.Component<
       overrides.Popover,
       Popover,
     );
+
+    if (__DEV__ && this.props['aria-label']) {
+      // eslint-disable-next-line no-console
+      console.warn(
+        `Providing i18n data through properties will be removed in the next major version. Please use the LocalProvider.`,
```

Comment:
Do we really want to remove the props? They are straightforward to use and can be handy for some tweaks and overrides even when the `LocalProvider` is used. Also, it's not a big overhead for us and seems like an unnecessary breaking change that requires a lot of grunt work to fix.

// The only thing I would change is to group them into a single `this.props.translate` object so it's consistent across all components and doesn't pollute the top-level props but that ship already sailed.

Reply:
That can work too - the idea behind it was to make sure we only have a single API interface to maintain (and make it straightforward from a consumer point of view too), but I guess we can go either way.

What do you think @nadiia @tajo @schnerd ?

Reply:
Let's keep the props as they are - I am going to update this PR

Reply:
Agree on keeping the aria attr in props 👍 For those who don't use i18n would be slightly too much work to do in order to change those if removed, specifically for an input case, for example.